tests/unit/projection-test.js

```diff
@@ -94,308 +140,328 @@ module('unit/projection', function(hooks) {
         }
       },
 
-      models: {
-        [NORM_BOOK_CLASS_PATH]: {
-          aliases: {
-            name: 'title',
-            cost: 'price',
-            pub: 'publisher',
-            releaseDate: 'pubDate',
-            pb: 'paperback',
-            hb: 'hardback',
-          },
-          defaults: {
-            publisher: 'Penguin Classics',
-            hardback: true,
-            paperback: true,
-            publishedIn: 'US',
-          },
-          transforms: {
-            // This interferes with the URN resolution
-            // publisher(value) {
-            //   return `${value}, of course`;
-            // },
-            pubDate(value) {
-              return new Date(Date.parse(value));
-            }
-          }
-        },
-        [NORM_BOOK_EXCERPT_PROJECTION_CLASS_PATH]: {
-          projectedType: NORM_BOOK_CLASS_PATH,
-          attributes: ['title', 'author', 'year', 'publisher'],
-        },
-        [NORM_BOOK_PREVIEW_PROJECTION_CLASS_PATH]: {
-          projectedType: NORM_BOOK_CLASS_PATH,
-          resolvedTypes: {
-            publisher: NORM_PROJECTED_PUBLISHER_CLASS,
-            author: NORM_PROJECTED_AUTHOR_CLASS
-          },
-          // if you want to project an embedded model then it must have a type
-          //  computedEmbeddedType
-          attributes: ['title', 'author', 'chapter-1', 'year', 'publisher'],
-        },
-        [PUBLISHER_CLASS]: {},
-        // this schema must come with the parent schema
-        [NORM_PROJECTED_AUTHOR_CLASS]: {
-          attributes: ['location', 'name']
-        },
-        [NORM_PROJECTED_PUBLISHER_CLASS]: {
-          projectedType: PUBLISHER_CLASS,
-          attributes: ['location', 'name']
```

Comment:
why is this moved to a closure var?

Reply:
because I didn't realize we bound the scope for methods in the schema and so configured it this way. The change happens because @dnachev also added in accessing of models in a separate PR which when rebased became a change vs an addition. I can revert the use of a closure var now that I see that schemas have a context.